docs/Gemfile.lock

```diff
@@ -30,18 +30,19 @@ GEM
       redcarpet (~> 3.1)
       safe_yaml (~> 1.0)
       toml (~> 0.1.0)
-    jekyll-coffeescript (1.0.1)
+    jekyll-coffeescript (1.0.2)
       coffee-script (~> 2.2)
+      coffee-script-source (~> 1.11.1)
```

Comment:
I find [https://github.com/jekyll/jekyll/pull/5662](https://github.com/jekyll/jekyll/pull/5662), that PR add the dependency.